src/components/combos/ComboDetails.js

```diff
@@ -188,7 +189,7 @@ const ComboDetails = ({ setUserCombo, comboToShow, addTrickToCombo }) => {
 
           {!inGenerator && !inPostCombo && (
             <div className="boostSkill row justify-content-center">
-              <button className={combo.boostSkill ? "col-8 col-lg-3 col-xl-2 btn btn-warning" : "col-8 col-lg-3 col-xl-2 btn btn-primary" } onClick={toggleBoostSkill}>{combo.boostSkill ? "Unboost this combo" : (<><IoRocketSharp/> Boost this combo</>)}</button>
+              <button className={combo.boostSkill ? "col-8 col-lg-3 col-xl-2 btn btn-warning" : "col-8 col-lg-3 col-xl-2 btn btn-primary" } onClick={toggleBoostSkill}>{combo.boostSkill ? "Unboost this combo" : (<><IoRocketSharp/> <Trans id="comboStats.boostCombo">Boost this combo</Trans></>)}</button>
```

Comment:
Here we still need to wrap "Unboost this combo" in a <Trans> tag

Reply:
Fixed in [334d334](https://github.com/bastislack/highline-freestyle/pull/198/commits/334d3346a6a6506656230c65abcaddba41153219)